<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompanyName - Transferencias</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivos">
    <property name="title">
     <string>Archivos</string>
    </property>
    <addaction name="btnRegistrarTransferencias"/>
    <addaction name="btnReportarTransferencia"/>
    <addaction name="btnHistorialTransferencias"/>
   </widget>
   <widget class="QMenu" name="menuCreditos">
    <property name="title">
     <string>Cr√©ditos</string>
    </property>
    <addaction name="actionDesarrolladores"/>
   </widget>
   <addaction name="menuArchivos"/>
   <addaction name="menuCreditos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDesarrolladores">
   <property name="text">
    <string>Desarrolladores</string>
   </property>
  </action>
  <action name="btnRegistrarTransferencias">
   <property name="text">
    <string>Registrar Transferencias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="btnReportarTransferencia">
   <property name="text">
    <string>Reportar Transferencia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="btnHistorialTransferencias">
   <property name="text">
    <string>Historial de Transferencias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
